roles/setup_gitea/templates/secret_gitea_app_ini.j2

```diff
@@ -0,0 +1,7 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: gitea-app-ini
+  namespace: "{{ sg_namespace }}"
+data:
+  app.ini: {{ lookup('template', 'gitea_app_ini.j2') | b64encode }}
```

Comment:
Missed line break at the end of the file

Reply:
Done